backend/api-requests.http

```diff
@@ -0,0 +1,18 @@
+# vscode extension "Humao Rest Client"
+
+@baseUrl = http://wandergold.local
+
+
+###
+
+GET {{baseUrl}}/products
+
+
+###
+
+POST {{baseUrl}}/products HTTP/1.1
+Content-Type: application/json
+
+{
+    "title": "Wanderung foo"
+}
```

Comment:
You could add this file to your .gitignore

Reply:
You're right, thank you!